delete_all_packages.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# This script allows you to mass delete all packages from a 'user' repository.
+# This only deletes 30 packages at a time. Repeat it until everything is deleted.
+
+import requests
+
+# Replace these variables with a Personal access token from GitHub
+your_token = 'ghp_'
+user = 'user'
```

Comment:
This is quite risky. You might commit your credentials by accident.

How about this:


```suggestion
import netrc

# Configure your GitHub user name and Personal Access Token (PAT) to the `.netrc` file. 
user, _, your_token = netrc.netrc().authenticators("api.github.com")
```